submissions/famuyiwaOlawumi/html-forms/htmlform-8.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>signup page</title>
+</head>
+<body>
+    <p>
+        <a href="htmlform-8.html">Signup</a>
+        <a href="../htmltask/index.html">Product page</a>
```

Comment:
navigation links is for task 8. Please remove them

Reply:
This is particular pull request for task 8 as the name implies ma'am. The instruction was to create pull request for each task. Nothing was said about waiting for one to be merged before creating another.
Thanks